test/ToricVarieties/affine_normal_varieties.jl

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
?

Reply:
Thx. This was there by accident. I just removed it.